CHANGELOG.rst

```diff
@@ -96,13 +105,23 @@ Interactive improvements
 - When executing a command, abbreviations are no longer expanded when the cursor is separated from the command by spaces, making it easier to suppress abbreviation expansion of commands without arguments. (:issue:`8423`).
 - ``fish_key_reader``'s output was simplified. By default, it now only prints a bind statement. The previous per-character timing information can be seen with a new ``--verbose`` switch (:issue:`8467`).
 - Custom completions are now also loaded for commands that contain tildes or variables like ``~/bin/fish`` or ``$PWD/fish`` (:issue:`8442`).
-- Command lines spanning multiple lines will not be overwritten by the completion pager when it fills the entire terminal (:issue:`8509`).
+- Command lines spanning multiple lines will not be overwritten by the completion pager when it fills the entire terminal (:issue:`8509`, :issue:`8405`).
+- When redrawing a multiline prompt, the old prompt was not fully cleared. This has been corrected (:issue:`8163`).
+- Interactive completion would occasionally ignore the last word on the command line due to a race condition. This has been fixed (:issue:`8175`).
+- fish would print errors when trying to autoload completions for command names with embedded null bytes, because those names are truncated by C library routines. This has been fixed by refusing to autoload commands with null bytes (:issue:`8195`).
+- When a short-lived fish process writes a universal variable, propagation was slow in some cases. This has been sped up (:issue:`8209`).
```

Comment:
How about "Propagation of universal variables from a fish process that is closing is faster"?